src/Cohesity.Powershell/Scripts/Set-CohesityAlertResolutions.ps1

```diff
@@ -22,20 +53,16 @@ function Set-CohesityAlertResolutions {
     }
 
     Process {
-        $url = $server + '/irisservices/api/v1/public/alertResolutions/'+$resolvedAlertId
-
-        $headers = @{'Authorization'='Bearer '+$token}
-        $payload = @{
-            alertIdList=$alertIds
+        if("" -eq $resolvedAlertId) {
```

Comment:
Cant we check if object is Null ? instead of empty string ?

Reply:
Fixed. Avoid using strong data type to leverage the usage of $null